levels/scripts.h

```diff
@@ -6,23 +6,42 @@
 
 // scripts
 extern const LevelScript level_main_scripts_entry[];
+#define common1_load_geolayouts script_func_global_1
```

Comment:
No, this is probably why the common0 thing wasn't working

Reply:
though it is the wrong name, I believe the one 2 lines down is the one that crashes

Reply:
iirc it's not safe to load those twice? (though idk why)